packages/expect/src/jasmine_utils.js

```diff
@@ -209,7 +209,12 @@ function keys(obj, isArray, hasKey) {
         keys.push(key);
       }
     }
-    return keys.concat((Object.getOwnPropertySymbols(o): Array<any>));
+    return keys.concat(
+      (Object.getOwnPropertySymbols(o): Array<any>).filter(
+        //$FlowFixMe Jest complains about nullability, but we know for sure that property 'symbol' does exist.
```

Comment:
It's probably mistyped in Flow libdefs. Would be great to send them a PR with a fix, but I have never done it yet, so I won't recommend anything 😅 cc @cpojer who is Flow expert recently ;)